yarn-project/noir-contracts/src/contracts/test_contract/src/main.nr

```diff
@@ -2,6 +2,7 @@
 contract Test {
```

Comment:
I'm trying to spot the pattern of what functionality is being tested in this contract. Seems like quite a random hodgepodge of features being tested in here. Not that this should be addressed in this PR. Just an observation.

Reply:
TIL "hodgepodge" and I loved it. I'll add a better comment to the file. And I may use hodgepodge on it.